_health-care/_js/_form.js

```diff
@@ -27,258 +53,258 @@ var HealthApp = (function() {
-    { node: "CHContrecordno",  arg: "CHILD #2 OF 2", transform: identityTransform },
-    { node: "ChildNameLast",  arg: "LastChild2", transform: identityTransform },
-    { node: "ChildNameFirst",  arg: "FirstChild2", transform: identityTransform },
-    { node: "ChildNameMiddle",  arg: "MiddleChild2", transform: identityTransform },
-    { node: "ChildNameSuffix",  arg: "II", transform: identityTransform },
-    { node: "ChildRelationshipSon",  arg: "0", transform: identityTransform },
-    { node: "ChildRelationshipDaughter",  arg: "1", transform: identityTransform },
-    { node: "ChildRelationshipStepson",  arg: "0", transform: identityTransform },
-    { node: "ChildRelationshipStepdaughter",  arg: "0", transform: identityTransform },
-    { node: "ChildSocialSecurityNumber",  arg: "555-44-3333", transform: identityTransform },
-    { node: "ChildDependentDateMonth",  arg: "4", transform: identityTransform },
-    { node: "ChildDependentDateDay",  arg: "5", transform: identityTransform },
-    { node: "ChildDependentDateYear",  arg: "1992", transform: identityTransform },
-    { node: "ChildDependentDate",  arg: "04/05/1992", transform: identityTransform },
-    { node: "ChildDateOfBirthMonth",  arg: "4", transform: identityTransform },
-    { node: "ChildDateOfBirthDay",  arg: "5", transform: identityTransform },
-    { node: "ChildDateOfBirthYear",  arg: "1992", transform: identityTransform },
-    { node: "ChildDateOfBirth",  arg: "04/05/1992", transform: identityTransform },
-    { node: "ChildTotalDisabledYes",  arg: "0", transform: identityTransform },
-    { node: "ChildTotalDisabledNo",  arg: "1", transform: identityTransform },
-    { node: "ChildAttendSchoolYes",  arg: "1", transform: identityTransform },
-    { node: "ChildAttendSchoolNo",  arg: "0", transform: identityTransform },
-    { node: "ChildSelfSchool",  arg: "9876.00", transform: identityTransform },
-    { node: "ChildSupportAmount",  arg: null, transform: identityTransform },
-    { node: "ChildLive",  arg: "Y", transform: identityTransform },
-    { node: "ChildLiveNo",  arg: "0", transform: identityTransform },
-    { node: "ChildProvideSupportYes",  arg: "0", transform: identityTransform },
-    { node: "ChildProvideSupportNo",  arg: "0", transform: identityTransform },
-    { node: "ChildGrossIncome",  arg: "54.00", transform: identityTransform },
-    { node: "ChildPropertyIncome",  arg: "43.00", transform: identityTransform },
-    { node: "ChildOtherIncome",  arg: "32.00", transform: identityTransform }
+    { node: "DateOfBirthMonth",  arg: 'veteran[date_of_birth]', transform: extractMonthTransform },
+    { node: "DateOfBirthDay",  arg: 'veteran[date_of_birth]', transform: extractDateTransform },
+    { node: "DateOfBirthYear",  arg: 'veteran[date_of_birth]', transform: extractYearTransform },
+    { node: "PlaceOfBirthCity",  name: 'veteran[]', arg: "Kent", transform: identityTransform },
```

Comment:
Note, I'm using the `name: 'veteran[]'` field as a tag to know what I've looked at and what I haven't. It also lets me do a regex to auto-generate the unittest auto-fill lines. They will go away when we're fully done here.